lua/nvim-tree/view.lua

```diff
@@ -224,7 +224,16 @@ function M.open()
   a.nvim_command("vsp")
   local move_to = move_tbl[M.View.side]
   a.nvim_command("wincmd "..move_to)
-  a.nvim_command("vertical resize "..M.View.width)
+
+  local width
+  if (vim.g.nvim_tree_width_as_percent == true) then
```

Comment:
no parens in lua conditions please :)

Reply:
Should that not be ` == 1`? Aren't all other option in nvim-tree `0/1` rather than `true/false`?